cernopendata/cold_storage/api.py

```diff
@@ -187,11 +187,13 @@ def send_email(req, emails):
             logger.error(f"Failed to send email to {emails}: {e}")
 
     @staticmethod
-    def create(record_id, subscribers):
+    def create(record_id, subscribers, availability):
```

Comment:
what about giving a default value to availability (and then maybe also to subscribers)? 
That way, the tests do not have to be modified